### PR TITLE
create drtb-hiv-referral case

### PR DESCRIPTION
**Pull request by nickpell**
@proteusvacuum @czue

### CODE REVIEW

**Review by czue — APPROVED** [review]
lgtm. will also let @proteusvacuum take a quick look since he knows the data models better

File: custom/enikshay/nikshay_datamigration/factory.py [hunk 1]
```diff
@@ -97,7 +98,11 @@ def get_case_structures_to_create(self):
         person_structure = self.get_person_case_structure()
         ocurrence_structure = self.get_occurrence_case_structure(person_structure)
         episode_structure = self.get_episode_case_structure(ocurrence_structure)
-        return [episode_structure]
+        if not self._outcome or self._outcome.hiv_status == 'unknown':
```
Comment (proteusvacuum) [inline custom/enikshay/nikshay_datamigration/factory.py]: this should also trigger when `hiv_status = reactive`

Comment (proteusvacuum) [inline custom/enikshay/nikshay_datamigration/factory.py]: if there is no outcome, do you set the hiv_status of the episode to `unknown`?

Comment (nickpell) [inline custom/enikshay/nikshay_datamigration/factory.py]: >this should also trigger when hiv_status = reactive

updated

Comment (nickpell) [inline custom/enikshay/nikshay_datamigration/factory.py]: > if there is no outcome, do you set the hiv_status of the episode to unknown?

yes

**Review by proteusvacuum — APPROVED** [review]

**Pull request merged**